src/app/home-page/home-logged-out/home.component.html

```diff
@@ -320,8 +320,10 @@ <h5 class="mb-0">
           <span>Discourse forum about helpful Dockstore topics.</span>
         </div>
       </div>
-      <div fxFlex.md="33" fxFlex.gt-md="33" fxFlex.lt-md="100">
-        <div class="my-3 twitter" #twitter></div>
+      <div class="mt-timeline">
```

Comment:
Same as other comment, you should probably move this and next line into app-mastodon-timeline component.